client/client_test.go

```diff
@@ -484,24 +486,68 @@ func concurrentIncrements(db *client.DB, t *testing.T) {
 			// Wait until the other goroutines are running.
 			wgStart.Wait()
 
-			if err := db.Txn(func(txn *client.Txn) error {
-				txn.SetDebugName(fmt.Sprintf("test-%d", i))
-
-				// Retrieve the other key.
-				gr, err := txn.Get(readKey)
-				if err != nil {
-					return err
+			if isMultiphase {
```

Comment:
this is conflating two unrelated tests. why not have a new test to exercise the new functionality?